src/pfe/portal/routes/imagePushRegistry.route.js

```diff
@@ -15,21 +15,41 @@ const { validateReq } = require('../middleware/reqValidator');
 
 const log = new Logger(__filename);
 
+/**
+ * Temp API Function to bypass current IDE design for testing
```

Comment:
FYI this is a temp API that is in place to workaround CW on Che because CW IDE is calling this API endpoint before creating projects as it does not consume the new API endpoints yet.